internal/pkg/cli/validate.go

```diff
@@ -521,9 +537,9 @@ func validateKey(val interface{}) error {
 	if err != nil {
 		return errDDBAttributeBadFormat
 	}
-	err = dynamoTableNameValidation(*attr.Name)
```

Comment:
Is `dynamoTableNameValidation` used anywhere? if not can we delete it as well?

Reply:
I think it is still in use in storage_init.go to validate DDB table name